src/Factory/RenderErrorListenerFactory.php

```diff
@@ -4,8 +4,8 @@
 
 namespace Laminas\ApiTools\ApiProblem\Factory;
 
-use Interop\Container\ContainerInterface;
 use Laminas\ApiTools\ApiProblem\Listener\RenderErrorListener;
+use Psr\Container\ContainerInterface;
```

Comment:
This change is a potential breaking change for downstream extending classes: to avoid that, we probably need to include `laminas/laminas-servicemanager` as a polyfill :thinking: